docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.3'
```

Comment:
For further convenience I added a docker-compose file. Otherwise the docker command would have become quite long, as it would have to specify the port mapping and volume mapping.

A volume is used to access the current docs at runtime. Otherwise you'd have to start the docker container for each change in the docs.

However, we don't have fancy features like hot reload. In order to see a change reflected in the running preview. Do this:

- Change a comment in a source file
- run docs-ts
- refresh the browser at 4000